compiler/dcalc/dune

```diff
@@ -2,6 +2,7 @@
  (name dcalc)
  (public_name catala.dcalc)
  (libraries bindlib unionFind utils re camomile runtime)
+ ;; (flags (:standard -rectypes))
```

Comment:
What's this for?

Reply:
Ah just an experiment I played with at some point, I'll remove the comment